src/Store/SessionStore.php

```diff
@@ -98,6 +101,10 @@ public function getMethod(): ?string
      */
     public function setMethod($method): StoreInterface
     {
+        if (in_array($method, $this->verifiedMethods)) {
```

Comment:
```suggestion
        if (in_array($method, $this->getVerifiedMethods())) {
```

(free code coverage)